packages/node/src/login/oidc/AggregateOidcHandler.ts

```diff
@@ -32,14 +32,19 @@ import {
   IOidcHandler,
   IOidcOptions,
   AggregateHandler,
+  ISessionInfo,
 } from "@inrupt/solid-client-authn-core";
+import fetch from "cross-fetch";
 
 /**
  * @hidden
  */
 @injectable()
 export default class AggregateOidcHandler
-  extends AggregateHandler<[IOidcOptions], void>
+  extends AggregateHandler<
+    [IOidcOptions],
+    (ISessionInfo & { fetch: typeof fetch }) | undefined
```

Comment:
Ditto - i.e. isn't this just the type `LoginResult`...?

Reply:
Fixed in 6ee8aac0c890e90d7e3af0fdf265de1327bb0a53